.github/workflows/docker.yml

```diff
@@ -0,0 +1,102 @@
+name: docker
+
+# FIXME: add yaml anchors when GitHub supports it,
+# strange that they don't
+
+on:
+  push:
+    # publish image as master=dev or on new tag
+    # except on document and ci changes
+    branches:
+      - master
+    tags:
+      - '*'
+    paths-ignore:
+      - '**.md'
+      - '.github/workflows/*yml'
+
+  # always run tests on merge
+  # except on document and ci changes
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '.github/workflows/*yml'
+
+  # schedule full rebuild and push on schedule, see todos
+  schedule:
+    - cron:  '13 3 * * *'
+
+env:
+  # TODO: remember to update version on new tag
+  LATEST_TAG: 2.4.2
```

Comment:
This looks like pycsw version. Did you also think about how to deal with the "package"/container versions? I.e., we might release 2.4.2 a couple of times before it's OK. Something like the release's versioning for distro packages: 2.4.2-1, 2.4.2-2...

Reply:
For this container I didn't consider that. I don't think we want to keep those versions around for posterity. As I see it dev should be used for those. Would it make any sense to have 

* `2.4.2-2`, `2.4`, `2`, `latest`
* `2.4.2-1`

I've not seen that done anywhere else with containers.

I think they fix the `2.4.2` version if they find a bug in the current container to ensure everyone get the correct version.

Reply:
I've seen it with the distro packages, not containers. Not sure if it's worth the trouble.